dev/appveyor-install-dependencies.ps1

```diff
@@ -26,7 +26,7 @@ Function InstallR {
   }
 
   $urlPath = ""
-  $latestVer = $(ConvertFrom-JSON $(Invoke-WebRequest http://rversions.r-pkg.org/r-release).Content).version
+  $latestVer = $(ConvertFrom-JSON $(Invoke-WebRequest http://rversions.r-pkg.org/r-release-win).Content).version
```

Comment:
I saw from http://rversions.r-pkg.org/r-release-win, it has a URL field: 

    "URL": "https://cran.r-project.org/bin/windows/base/R-3.4.1-win.exe"

If we go to use this URL, maybe we don't need to combine `rurl` below?